benchmarks/bench_sdbaseline.py

```diff
@@ -0,0 +1,643 @@
+
+import contextlib
+import filecmp
+import glob
+import os
+import shutil
+
+from casatools import ctsys, table
+from casatasks import sdbaseline
+from casatasks.private.sdutil import table_manager
+from casatasks.private.task_sdbaseline import check_fftthresh, is_empty, parse_wavenumber_param
+from casatestutils import selection_syntax
+
+
+tb = table()
+ctsys_resolve = ctsys.resolve
+
+# ASV iteration control (https://asv.readthedocs.io/en/stable/benchmarks.html#benchmark-attributes)
+number = 1            # i.e., always run the setup and teardown methods
+repeat = (1, 2, 30.0) # between 1 and 2 iterations per round w/ soft cutoff (start no new repeats) past 1m
+rounds = 3            # amount of instances a "repeat block" is run to collect samples
+min_run_count = 5     # enforce the min_repeat * rounds setting is met
+timeout = 3600        # conservative 1hr hard cap for duration of a single test execution
+
+def remove_single_file_dir(filename):
+    """
+    Remove a single file or a single directory.
+    For filename, '.' and those end with '..' (namely, '..', '../..' etc.)
+    are not allowed.
+    """
+    if filename == '.' or filename[-2:] == '..':
+        raise Exception("Caution! Attempting to remove '" + filename + "'!!")
+    
+    if os.path.exists(filename):
+        if os.path.isdir(filename):
+            shutil.rmtree(filename)
+        else: # file or symlink
+            os.remove(filename)
+
+
+def remove_files_dirs(filename):
+    """
+    Remove files/directories/symlinks 'filename*'.
+    For filename, '', '.' and those end with '..' (namely, '..', '../..' etc.)
+    are not allowed.
+    """
+    if filename == '.' or filename[-2:] == '..':
+        raise Exception("Caution! Attempting to remove '" + filename + "*'!!")
+    elif filename == '':
+        raise Exception("The parameter 'filename' must not be a null string.")
+    
+    filenames = glob.glob('{}*'.format(filename.rstrip('/')))
+
+    for filename in filenames:
+        remove_single_file_dir(filename)
+
+#def _get_num_files_by_keyword(cmpresult, keyword):
+#    return len(sum([eval(s[s.index(':')+1:]) for s in cmpresult if s.startswith(keyword)], []))
+
+
```

Comment:
It doesn't look like this commented out function gets included under the "code" key in `benchmarks.json` which I was worried about at first glance since none of these new tests are explicitly setting their `version` parameter. However it's still probably best just to delete this if it's not being used.